runtime/sanka/lang/System.c

```diff
@@ -2,6 +2,10 @@
 #include <stdio.h>
 #include <string.h>
 
+void System__print(char *text) {
+    fprintf(stdout, "%s", text);
```

Comment:
Use fputs() for consistency with System__println(). fputs() does not add a newline.